### DISCUSSION TITLE
Which is the proper branch for v3.1.0 release

### DISCUSSION THREAD

**Discussion by fh-dakselrod** [dotnet/SqlClient#1587] [Q&A] [answered]
When I check out `v3.1.0 branch`, I am getting some weirdness...
1. In tools/props/Versions.props I see:
`    <NugetPackageVersion Condition="'$(NugetPackageVersion)' == ''">5.0.0-dev</NugetPackageVersion>`
2. In git log I see `v5.0.0-preview1`

Is the 77ad58b1e0177ec282ba0e1a4e27a362fc330caf hash the right one for 3.1.0 or did it get messed up with others ?

**Comment by JRahnama** [comment, marked as answer]
@fh-dakselrod for 3.1 you can get it from servicing branch of 3.1
![image](https://user-images.githubusercontent.com/32077152/164274032-3f8064f0-fb47-40dc-9436-93240304966c.png)
that version inside the versionprops is for development and if you try to build the driver the generated nuget will have that version. You can easily change that value to any other number you want for development purposes.
